pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1856,7 +1856,14 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
           throw new Error('No container engine is set');
         }
         if (kubernetesVersion) {
-          await this.kubeBackend.install(config, kubernetesVersion, this.#adminAccess);
+          if (!config.kubernetes.version) {
+            const versionSetting = { kubernetes: { version: kubernetesVersion.version } };
+
+            await this.kubeBackend.install(merge(config, versionSetting), kubernetesVersion, this.#adminAccess);
+            mainEvents.emit('settings-write', versionSetting);
```

Comment:
It looks like we're already attempting to update the version:
https://github.com/rancher-sandbox/rancher-desktop/blob/e049562bc8b38e31956f4926f41e36ab99378168/pkg/rancher-desktop/backend/kube/lima.ts#L313
Is there a reason that's not working? (Should we delete that code, if we expect this to take over?)

Reply:
Oh, I'll try pulling out the `settings-write` part. The error happens in `install`, where it doesn't get the `kubernetesVersion` field and complains when `config.kubernetes.version` is empty

Reply:
And it does get set eventually.

Reply:
And better to put the version-merge code down the call stack where it's needed, rather than here.